src/mockhsm/commands.rs

```diff
@@ -303,6 +305,19 @@ fn get_object_info(state: &State, cmd_data: &[u8]) -> ResponseMessage {
     }
 }
 
+/// Get bytes of random data
+fn get_pseudo_random(_state: &State, cmd_data: &[u8]) -> ResponseMessage {
+    use std::iter;
+
+    let command: GetPseudoRandomCommand = deserialize(cmd_data)
+        .unwrap_or_else(|e| panic!("error parsing CommandType::GetPseudoRandom: {:?}", e));
+
+    let bytes = iter::repeat(4u8) // chosen by fair dice roll.
```

Comment:
I've generally been trying to mimic actual HSM behavior in the MockHSM for the most part...

Reply:
Can I take a dependency on rand to make that work, or is there another rng included already?